### PR TITLE
Add vSphere Volume Examples

### PR DESCRIPTION
**Pull request by abrarshivani**
This PR adds vSphere Volume Examples

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/34996)

### CODE REVIEW

File: examples/volumes/vsphere/README.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+      Capacity:	2Gi
+      Message:
+      Source:
+          Type:	vSphereVolume (a Persistent Disk resource in vSphere)
+          VolumePath:	[datastore1] kubevols/kubernetes-dynamic-pvc-80f7b5c1-94b6-11e6-a24f-005056a79d2d.vmdk
+          FSType:	ext4
+      No events.
+      ```    
+
+      __Note: VMDK is created inside kubevols folder in datastore which is mentioned in 'vsphere' cloudprovider config.
```
Comment (pdhamdhere) [inline examples/volumes/vsphere/README.md]: kubevols => `kubevols`

File: examples/volumes/vsphere/vsphere-volume-pvcsc.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: pvcsc001
+  annotations:
+    volume.beta.kubernetes.io/storage-class: slow
```
Comment (pdhamdhere) [inline examples/volumes/vsphere/vsphere-volume-pvcsc.yaml]: slow: pvscisi with zeroedthicj should be "fast" ;-)

**Review by pdhamdhere — COMMENTED** [review]
LGTM.

**Review by pdhamdhere — APPROVED** [review]
Thanks for doing this, Abrar. LGTM.

**Comment by abrarshivani** [comment]
//cc @dagnello @abithap @imkin @kerneltime @luomiao

**Comment by bgrant0607** [comment]
cc @kubernetes/examples @kubernetes/sig-storage

File: examples/volumes/vsphere/README.md [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+     See example
+
+     ```yaml
+        apiVersion: v1
+        kind: Pod
+        metadata:
+          name: test-vmdk
+        spec:
+          containers:
+          - image: gcr.io/google_containers/test-webserver
```
Comment (rootfs) [inline examples/volumes/vsphere/README.md]: if possible, provide the image tag

File: examples/volumes/vsphere/README.md [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+      Name:		pvcsc001
+      Namespace:	default
+      Status:		Bound
+      Volume:		pvc-80f7b5c1-94b6-11e6-a24f-005056a79d2d
+      Labels:		<none>
+      Capacity:	2Gi
+      Access Modes:	RWO
+      No events.
+      ```
+      Persistent Volume is automaically created and is bounded to this pvc.
```
Comment (rootfs) [inline examples/volumes/vsphere/README.md]: automatically

File: examples/experimental/persistent-volume-provisioning/README.md [hunk 5]
```diff
@@ -89,10 +89,10 @@ parameters:
 kind: StorageClass
 apiVersion: storage.k8s.io/v1beta1
 metadata:
-  name: slow
+  name: fast
```
Comment (rootfs) [inline examples/experimental/persistent-volume-provisioning/README.md]: keep slow, it is used by `claim1.json` later in the README.

**Comment by wattsteve** [comment]
Awesome docs. Thanks so much for contributing!

File: examples/volumes/vsphere/README.md (hunk 3 above)
Comment (abrarshivani) [inline examples/volumes/vsphere/README.md]: I cannot find tag for this image other than ```latest```.

**Comment by pdhamdhere** [comment]
@childsb can this be merged?

**Comment by kerneltime** [comment]
LGTM

**Comment by kerneltime** [comment]
:lgtm:

---

Review status: 0 of 10 files reviewed at latest revision, 5 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/34996#-:-KUrfFrdHWrPZcksbN_0:bnfp4nl)*

**Comment by abrarshivani** [comment]
@childsb @rootfs can this be merged?

**Comment by jsafrane** [comment]
@k8s-bot ok to test

**Comment by jsafrane** [comment]
From `make verify`:
```
I1025 00:35:44.808] 
I1025 00:35:44.810] /go/src/k8s.io/kubernetes/examples/ is out of date. Please run hack/update-munge-docs.sh
I1025 00:35:44.811] FAILED   hack/make-rules/../../hack/verify-munge-docs.sh	12s
```

**Comment by jsafrane** [comment]
lgtm

**Pull request merged**